content/en/docs/setup/production-environment/windows/user-guide-windows-nodes.md

```diff
@@ -143,21 +142,39 @@ Once you have a Linux-based Kubernetes master node you are ready to choose a net
     kubectl get pods --all-namespaces
     ```
 
-    ![alt_text](../flannel-master-kubeclt-get-pods.png "flannel master kubectl get pods screen capture")
+    The output looks like as follows:
+
+    ```shell
```

Comment:
nit: This isn't shell output. Leave the format code out and it'll work fine.

Reply:
Okay, I'll apply.